README.md

```diff
@@ -1,24 +1,35 @@
-# Lance
+<div align="center">
+<p align="center">
+ 
+<img width="257" alt="Lance" src="https://user-images.githubusercontent.com/917119/199353423-d3e202f7-0269-411d-8ff2-e747e419e492.png">
 
-**Blazing fast exploration and analysis of machine learning visual data using SQL**
+**Blazing fast exploration and analysis of computer vision data using SQL and DuckDB**
 
-```sql
-SELECT predict(‘resnet’, image) FROM dataset
-```
+<a href="https://eto-ai.github.io/lance/">Documentation</a> •
+<a href="https://blog.eto.ai/">Blog</a> •
+<a href="https://discord.gg/zMM32dvNtd">Discord</a> •
+<a href="https://twitter.com/etodotai">Twitter</a>
+
+![CI](https://github.com/eto-ai/lance/actions/workflows/cpp.yml/badge.svg)
+[![Docs](https://img.shields.io/badge/docs-passing-brightgreen)](https://eto-ai.github.io/lance/)
+![Python versions](https://img.shields.io/pypi/pyversions/pylance)
+ 
+<img width="600" alt="lance_splash_" src="https://user-images.githubusercontent.com/917119/199364478-b3f1e3cc-6462-4550-b55f-20a49158e82c.png">
 
-![CI](https://github.com/eto-ai/lance/actions/workflows/cpp.yml/badge.svg) [![Docs](https://img.shields.io/badge/docs-passing-brightgreen)](https://eto-ai.github.io/lance/) [![PyPi](https://img.shields.io/pypi/v/pylance)](https://pypi.org/project/pylance/) ![Python versions](https://img.shields.io/pypi/pyversions/pylance) [![](https://dcbadge.vercel.app/api/server/zMM32dvNtd?style=flat)](https://discord.gg/zMM32dvNtd)
+</p>
+</div>
 
-Lance makes machine learning workflows with visual data easy (images, videos, point clouds, audio, and more), by allowing Developers, Analysts and Operations to:
+<hr />
 
-* Use arbitary ML functions in SQL for common use cases such as similarity search using embeddings, model inference and computing evaluation metrics like F1, IOU and more.
+Lance makes machine learning workflows with computer vision data easy (images, videos, point clouds, audio, and more), by allowing Developers, Analysts and Operations to:
 
-* [Coming soon] Visualize, slice and drill-into visual datasets to inspect embeddings, labels/annotations, metrics and more.
+* Use SQL to greatly simplify common operations on computer vision data, such as similarity search for data discovery, model inference and computing evaluation metrics.
 
-* [Coming soon] Version, compare and diff visual datasets easily.
+* Visualize, slice and drill-into computer vision datasets to inspect embeddings, labels/annotations and metrics.
```

Comment:
Should we still indicate these features as in roadmap, not finished yet?

Reply:
Good point, will edit